src/index.ts

```diff
@@ -94,23 +94,41 @@ class Monika extends Command {
 
     // Read the config
     const file = flags.config
-    const config: Config = await parseConfig(file)
+    const config = await parseConfig(file)
+    const { notifications, probes } = config
+
+    // warn if config is empty
+    if ((notifications?.length ?? 0) === 0) {
+      const NO_NOTIFICATIONS_MESSAGE = `Notifications config has not been set. We will not be able to notify you when INCIDENT happened!\nPlease refer to Monika documentation for setting the notifications config at https://hyperjumptech.github.io/monika/guides/notifications.`
```

Comment:
Notifications has not been set. We will not be able to notify you when an INCIDENT occurs!\nPlease refer to the Monika documentations on how to configure notifications at https://hyperjumptech.github.io/monika/guides/notifications.`

Reply:
Fixed.